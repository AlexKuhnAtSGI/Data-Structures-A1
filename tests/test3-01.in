a
man
is
at
odds
to
know
his
mind,
because
his
mind
is
all
he
has
to
know
it
with
OOOHH IM HERE TO RUIN EVERYTHING









































shoo shoo ruiner