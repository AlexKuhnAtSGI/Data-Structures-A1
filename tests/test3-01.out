a
man
is
at
odds
to
know
his
mind,
because
his
mind
is
all
he
has
to
know
it
with
